apps/ledger-live-desktop/src/renderer/analytics/hooks/useTrackSyncFlow.test.ts

```diff
@@ -0,0 +1,156 @@
+import { renderHook } from "tests/testUtils";
+import { useTrackSyncFlow, UseTrackSyncFlow } from "./useTrackSyncFlow";
+import { track } from "../segment";
+import { UserRefusedAllowManager, UserRefusedOnDevice } from "@ledgerhq/errors";
+import { HOOKS_TRACKING_LOCATIONS } from "./variables";
+
+jest.mock("../segment", () => ({
+  track: jest.fn(),
+  setAnalyticsFeatureFlagMethod: jest.fn(),
+}));
+
+describe("useTrackSyncFlow", () => {
+  const deviceMock = {
+    modelId: "Europa",
+    wired: true,
+  };
+
+  const defaultArgs: UseTrackSyncFlow = {
+    location: HOOKS_TRACKING_LOCATIONS.ledgerSync,
+    device: deviceMock,
+    allowManagerRequested: null,
+    error: null,
+    isLedgerSyncOpenAppRequested: false,
+    isLedgerSyncAppOpen: false,
+    isTrackingEnabled: true,
+  };
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should track 'Secure Channel approved' when allowManagerRequested changes from true to false without error", () => {
+    const { rerender } = renderHook((props: UseTrackSyncFlow) => useTrackSyncFlow(props), {
+      initialProps: {
+        ...defaultArgs,
+        allowManagerRequested: true,
+      },
+    });
+
+    rerender({
+      ...defaultArgs,
+      allowManagerRequested: false,
+    });
+
+    expect(track).toHaveBeenCalledWith(
+      "Secure Channel approved",
+      expect.objectContaining({
+        deviceType: "Europa",
+        connectionType: "USB",
```

Comment:
Shouldn't we use `CONNECTION_TYPES.USB` everywhere instead of `"USB"` ? Same for `"BLE"`